$OpenBSD: patch-pllua_h,v 1.2 2013/10/15 02:21:13 jeremy Exp $

Recent versions of PostgreSQL require an extra header to get the
Relation struct defintion, and pllua hasn't been updated recently.

--- pllua.h.orig	Sun Sep 20 07:22:21 2009
+++ pllua.h	Thu Oct  3 13:17:41 2013
@@ -11,6 +11,7 @@
 #include <fmgr.h>
 #include <funcapi.h>
 #include <access/heapam.h>
+#include "access/htup_details.h"
 #include <catalog/namespace.h>
 #include <catalog/pg_proc.h>
 #include <catalog/pg_type.h>
@@ -25,6 +26,7 @@
 #include <utils/datum.h>
 #include <utils/builtins.h>
 #include <utils/array.h>
+#include <utils/rel.h>
 /* Lua */
 #include <lua.h>
 #include <lualib.h>
